Create ospool_for_education.md (#94)
osg-htc/user-documentation: documentation/support_and_training/training/ospool_for_education.md

---
ospool:
  path: support_and_training/training/ospool_for_education.md
---

OSPool Resources for Teaching & Education
====================================

> The OSPool provides a ***free, ready-to-use*** platform for instructors to use when teaching high throughput computing concepts for academic courses, conference workshops, and other events. 

Reviewer: ```suggestion
The OSPool provides a ***free, ready-to-use*** platform for instructors who are teaching high throughput computing concepts for academic courses, conference workshops, and other events. 
```

Instructors can choose for their students to have Guest or Full Accounts on the OSPool. For Guest Accounts, students/attendees can launch an OSPool Notebook at any time and practice job submission with smaller workflows. For Full Accounts, students/attendees will need to request an account (which will be approved within one business day), but then are able to submit large scale high throughput computing workflows free of charge. 

> See the table below to learn more about how to use OSPool resources for your event.

Reviewer: Maybe instead "The table below outlines suggested steps for bringing OSPool resources to your training or event. Please reach out to the facilitation team at any time if you have questions or want to chat about your goals."

Reviewer: I just want it to be clear that these are steps, because they aren't numbered.

> See the table below to learn more about how to use OSPool resources for your event.

Author: ```suggestion
The table below outlines suggested steps for bringing OSPool resources to your training or event. Please reach out to the facilitation team at any time if you have questions or want to chat about your goals.
```

<table>
  <tr>
      <th>Explore our Tools</th>
      <td colspan="2"><b>Explore HTCondor job submission with a OSPool Guest Account</b>
        <br>

>         - To launch a guest account, go to <a href="https://notebook.ospool.osg-htc.org">https://notebook.ospool.osg-htc.org</a> using an internet browser.

Reviewer: ```suggestion
        - To launch a guest OSPool Notebook, go to <a href="https://notebook.ospool.osg-htc.org">https://notebook.ospool.osg-htc.org</a> using an internet browser. 
```

        <br>
        - Visit our <a href="https://portal.osg-htc.org/documentation/htc_workloads/submitting_workloads/jupyter/">OSPool Jupyter Notebooks guide</a> to learn about Guest and Full Accounts
        <br>
      </td>
  </tr>
  <tr>
      <th>Conduct Initial Testing of your Event Materials</th>
      <td colspan="2"><b>Using the Guest Account, we recommend conducting initial testing of your event materials to help inform next steps.</b>
        <br>
        <br>

>          We provide supplementary materials that you may use to help teach high throughput computing and OSPool-related concepts.  

Reviewer: link to end of guide?

>          We provide supplementary materials that you may use to help teach high throughput computing and OSPool-related concepts.  

Author: ```suggestion
         We provide supplementary materials <a href="#supplementary-resources">supplementary materials</a> that you may use to help teach high throughput computing and OSPool-related concepts.  
```

      </td>
  </tr>
  <tr>
    <th>Discuss your event goals with a Research Computing Facilitator (Optional)</th>

>     <td colspan="2">The Facilitation team is here to help discuss your event goals and guide you through the process of using OSG resources. Fill out this form {LINK} and a Facilitator will contact you within one business day about scheduling a short virtual meeting.

Reviewer: ```suggestion
    <td colspan="2">The Facilitation team is here to help discuss your event goals and provide guidance about how to best leverage existing OSG services and resources. Fill out this form {LINK} and a Facilitator will contact you within one business day about scheduling a short virtual meeting.
```

    </td>
  </tr>
  <tr>
    <th>Evaluate Guest or Full Account for Attendees</th>
    <td><b>Option 1: Guest OSPool Accounts</b>
      <br>
      <br>
      You are welcome have your attendees use an OSPool Guest Acccount for the event.
      <br>
      <br>
      This is a good option for events that:
      <br>
      <br>
      - may not know registrants in advance 
      <br>
      - run less than 4 hours, or can easily recreate files that may be lost upon session time out (4 hours)
      <br>

>       - would like jobs to access the full capacity of the OSPool

Reviewer: ```suggestion

```

      <br>
      - want to only use a notebook interface
    </td>
        <td><b>Option 2: Full OSPool Accounts</b>
      <br>
      <br>
      You can request that your students have the ability to submit jobs to the OSPool using Full Accounts. 
      <br>
      <br>
      This is a good option for events that:
      <br>
      <br>
      - know registrants in advance 
      <br>
      - will run for more than 1-2 days
      <br>
      - with more than 50 participants

>       <br>

Reviewer: ```suggestion
      <br>
      - would like jobs to access the full capacity of the OSPool
```

      - would like to submit jobs using a notebook or classic terminal interface
    </td>
  </tr>
  <tr>
    <th>Prior to the Event</th>
    <td colspan="2"> 

>       Participants requesting full accounts should request an account a few days in advance of the event (does not apply to guest accounts). It is also good practice to test your full workshop code and any software on your account of choice. 

Reviewer: ```suggestion
      If using full accounts, the instructor will provide a list of participants to the OSG Research Facilitation Team, and participants should request an account a few days in advance of the event (does not apply to guest accounts). It is also good practice to test your full workshop code and any software on your account of choice. 
```

    </td>
  </tr>
    <th>Start of Event</th>
    <td colspan="2">We require all events to provide a short (~5 minute) <a href="https://docs.google.com/presentation/d/1T-n7CkMKftjglfm2aD6iVCCMMOknbpjP/edit#slide=id.p1">introduction to OSG Policies</a>.</td>
  <tr>
    <th>Feedback</th>

>     <td colspan="2">After your event, please email us to let us know how it went.

Reviewer: ```suggestion
    <td colspan="2">After your event, please email us to let us know how it went, and the number of participants. 
```

    </td>
  </tr>
</table>

> ## Teaching Resources

Author: ```suggestion
## Teaching Resources
<a id="supplementary-resources">Here are some resources you can use for your event:</a>
```

### Worksheets for Public Use
<a href="https://docs.google.com/presentation/d/1_7d-qW7uRV-lPJ9iYqt8JPVacXYkY0k_aocoFzwXlzY/edit#slide=id.p">Scale Out My Computing Brainstorming Worksheet</a>

### Slide Presentations for Public Use
<a href="https://docs.google.com/presentation/d/1T-n7CkMKftjglfm2aD6iVCCMMOknbpjP/edit#slide=id.p1">OSG Policies and Intro for Course Use</a>

<a href="https://portal.osg-htc.org/documentation/support_and_training/training/osgusertraining/">OSPool Training Slides and Recordings</a>

### Video Recordings
<a href="https://portal.osg-htc.org/documentation/support_and_training/training/osgusertraining/">OSPool Training Slides and Recordings</a>

<a href="https://www.youtube.com/watch?v=oMAvxsFJaw4&list=PLO7gMRGDPNumCuo3pCdRk23GDLNKFVjHn">HTCondor User Tutorials</a>

> 

Reviewer: Link to the OSG/OSG Connect channel here as well?

> 

Author: ```suggestion

<a href="https://www.youtube.com/@partnershiptoadvancethroug3036">Partnership to Advance Throughput Computing YouTube channel</a>
```

<table>
  <tr>
      <th>Explore our Tools</th>
      <td colspan="2"><b>Explore HTCondor job submission with a OSPool Guest Account</b>
        <br>
        - To launch a guest OSPool Notebook, go to <a href="https://notebook.ospool.osg-htc.org">https://notebook.ospool.osg-htc.org</a> using an internet browser. 
        <br>
        - Visit our <a href="https://portal.osg-htc.org/documentation/htc_workloads/submitting_workloads/jupyter/">OSPool Jupyter Notebooks guide</a> to learn about Guest and Full Accounts
        <br>
      </td>
  </tr>
  <tr>
      <th>Conduct Initial Testing of your Event Materials</th>
      <td colspan="2"><b>Using the Guest Account, we recommend conducting initial testing of your event materials to help inform next steps.</b>
        <br>
        <br>
         We provide supplementary materials <a href="#supplementary-resources">supplementary materials</a> that you may use to help teach high throughput computing and OSPool-related concepts.  
      </td>
  </tr>
  <tr>
    <th>Discuss your event goals with a Research Computing Facilitator (Optional)</th>

>     <td colspan="2">The Facilitation team is here to help discuss your event goals and provide guidance about how to best leverage existing OSG services and resources. Fill out this form {LINK} and a Facilitator will contact you within one business day about scheduling a short virtual meeting.

Author: ```suggestion
    <td colspan="2">The Facilitation team is here to help discuss your event goals and provide guidance about how to best leverage existing OSG services and resources. Fill out <a href="https://portal.osg-htc.org/event-request">this form</a> and a Facilitator will contact you within one business day about scheduling a short virtual meeting.
```
